tools-v2/pkg/config/bs.go

```diff
@@ -47,6 +47,7 @@ const (
 	CURVEBS_PASSWORD           = "password"
 	VIPER_CURVEBS_PASSWORD     = "curvebs.root.password"
 	CURVEBS_DEFAULT_PASSWORD   = "root_password"
+	CURVEBS_DIR                = "dir"
```

Comment:
Increase VIPER_CURVEBS_DIR, and add mapping relationship in BSFLAG2VIPER, so that the configuration file can also be read when the configuration file is provided.